Detections/3pNetworkDetetions/PhishiglinkExecutionObserved.yaml

```diff
@@ -0,0 +1,83 @@
+id: 2fed0668-6d43-4c78-87e6-510f96f12145
+name: Phishing link click observed in 3P Network device
+description: |
+  'sucessfull Phishing link accessed via "Palo Alto Networks","Fortinet","Check Point","Zscaler" devices.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId:  OfficeATP
+    dataTypes:
+    - OfficeATP
+    - Office365
+    - CommonSecurityLog
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
```

Comment:
This doesn't map to the scenario we talked about here - we are looking for phishing attacks here - how is this mapping to C2?

Reply:
@petebryan, This should be executed while the user executes the URL and the entity has the c2 URL, I am not sure how we can add that conditions here, Please share the Examples if we have any.